### PR TITLE
Add CRL checking for Apache 2.4 to Puppet vhost

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

File: spec/classes/puppet_server_spec.rb [hunk 1]
```diff
@@ -46,6 +46,7 @@
         should contain_class('puppet::server::passenger').
           with_ssl_cert('/var/lib/puppet/ssl/certs/puppetmaster.example.com.pem').
           with_ssl_cert_key('/var/lib/puppet/ssl/private_keys/puppetmaster.example.com.pem')
```
Comment (ekohl) [inline spec/classes/puppet_server_spec.rb]: missing a dot here

**Comment by ekohl** [comment]
:+1: once the tests pass

File: spec/classes/puppet_server_spec.rb (hunk 1 above)
Comment (domcleal) [inline spec/classes/puppet_server_spec.rb]: many thanks!

**Comment by mmoll** [comment]
Working fine on CentOS/7 and Debian/wheezy.
Merged as de4856d, thanks! :white_check_mark:

**Pull request closed without merging**